src/Tribe/Organizer.php

```diff
@@ -226,6 +227,19 @@ public function linked_post_type_container( $container, $post_type ) {
 	}
 
 	/**
+	 * Removes anything other than integers from the supplied array of Organizer IDs.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $organizer_ids An array of post IDs of the current event's attached Organizers.
+	 * @param int $post_id The current event's post ID.
+	 * @return array
```

Comment:
Adding an extra line before `@return` to separate it from the parameters would be primo and in line with WP coding standards.